.github/workflows/safari_stable.yml

```diff
@@ -0,0 +1,87 @@
+name: "All Tests: Safari (stable)"
+
+# We never interact with the GitHub API, thus we can simply disable all
+# permissions the GitHub token would have.
+permissions: {}
+
+on:
+  push:
+    branches:
+      - epochs/daily
+      - triggers/safari_stable
+
+env:
+  safaridriver_diagnose: false
+
+jobs:
+  results_safari:
+    name: "All Tests: Safari (stable)"
+    runs-on:
+      - self-hosted
+      - webkit-ews
+    timeout-minutes: 180
+    strategy:
+      matrix:
+        current-chunk: [1, 2, 3, 4, 5, 6, 7, 8]
+        total-chunks: [8]
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4.1.0
+        fetch-depth: 1
+      - name: Enable safaridriver diagnostics
+        if: env.safaridriver_diagnose == true
+        run: |-
+          rm -rf ~/Library/Logs/com.apple.WebDriver/
+          defaults write com.apple.WebDriver DiagnosticsEnabled 1
+      - name: Enable safaridriver
+        run: |-
+          set -eux -o pipefail
+          export SYSTEM_VERSION_COMPAT=0
```

Comment:
I guess this is harmless, but wasn't this a workaround for something quite long ago that's no longer needed?

Reply:
Oh, for `safaridriver` this absolutely shouldn't be needed.

For actually running tests, I believe this depends on what SDK the binaries we're running (primarily the Python interpreter) are built against — if they're built against a pre-macOS 11 SDK, then without this macOS will claim to be 10.16.

Given this setup is using the Xcode-provided Python, we can probably entirely get rid of this.